ladybug_geometry/geometry3d/cylinder.py

```diff
@@ -0,0 +1,188 @@
+# coding=utf-8
+"""Cylinder"""
+from __future__ import division
+
+from .pointvector import Point3D, Vector3D
+
+import math
+
+
+class Cylinder(object):
+    """Cylinder object.
+
+    Args:
+        center: A Point3D at the center of the bottom base of the cylinder.
+        axis: A Vector3D representing the direction and height of the cylinder.
+            The vector extends from the bottom base center to the top base center.
+        radius: A number representing the radius of the cylinder.
+
+    Properties:
+        * center
+        * axis
+        * radius
+        * diameter
+        * height
+        * area
+        * volume
```

Comment:
I like how you have set up the init method of this object but I think we will need easy access to the `center_end` point in order to use this for certain applications. This should be easy to add as a property since it's effectively the following:
```
self.center + self.axis
```
Along these lines, it would be good to have a classmethod to initialize the cylinder from a start point, end point, and radius since I know that this is how cylinders are represented in the Radiance engine.